Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -191,4 +196,31 @@ namespace AudioCommon
 		g_sound_stream->GetMixer()->StopLogDSPAudio();
 		s_audio_dump_start = false;
 	}
+
+	void IncreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
+		int& currentVolume = SConfig::GetInstance().m_Volume;
+		currentVolume += offset;
+		if (currentVolume > AUDIO_VOLUME_MAX)
+			currentVolume = AUDIO_VOLUME_MAX;
+		UpdateSoundStream();
+	}
+
+	void DecreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
+		int& currentVolume = SConfig::GetInstance().m_Volume;
+		currentVolume -= offset;
+		if (currentVolume < AUDIO_VOLUME_MIN)
+			currentVolume = AUDIO_VOLUME_MIN;
```

Comment:
This will behave differently from what the UI allows you to do. The UI doesn't allow you to ever get lower than 1% (thus never actually muting) while this allows you to get to 0%. It's a bit odd to have the UI state it's at 0% but it actually can't be set to 0% without the keybinding. Do we want to change the UI to allow 0% or do we set this minimum to 1%?